lib/routes/redirect.js

```diff
@@ -6,11 +6,21 @@ const url = require('url');
 
 const config = require('../config');
 
+function actionToPathname(action) {
+  if (action === 'signup') {
+    return 'signup';
+  } else if (action === 'force_auth') {
+    return 'force_auth';
+  }
+
+  return 'signin';
```

Comment:
Should we consider erroring out if an invalid `action` is specified?  I can imagine people accidentally typoing this as e.g. `force_signin`.